sdk/servicebus/azure-servicebus/tests/async_tests/mgmt_tests/test_mgmt_topics_async.py

```diff
@@ -88,6 +89,78 @@ async def test_async_mgmt_topic_create_with_topic_description(self, servicebus_n
             assert topic_2.enable_express
             assert topic_2.enable_partitioning
             assert topic_2.max_size_in_megabytes % 3072 == 0
+
+            with pytest.raises(HttpResponseError):
+                await mgmt_service.create_topic(
+                    topic_name_3,
+                    max_message_size_in_kilobytes=10
```

Comment:
same question: is this needed here when min size is being tested below?

Reply:
same, this is testing that standard tier doesn't support `max_message_size_in_kilobytes`, I should update the test case not to use 10 which is misleading as if I were testing the min threshold.

Reply:
test updated to 1024